Framework/src/runBasic.cxx

```diff
@@ -85,47 +85,48 @@ WorkflowSpec defineDataProcessing(const ConfigContext& config)
     "producer",
     Inputs{},
     Outputs{
-      { "ITS", "RAWDATA", 0, Lifetime::Timeframe }
-    },
+      { "ITS", "RAWDATA", 0, Lifetime::Timeframe } },
     AlgorithmSpec{
-      (AlgorithmSpec::InitCallback) [](InitContext&) {
+      (AlgorithmSpec::InitCallback)[](InitContext&){
         std::default_random_engine generator(11);
-        return (AlgorithmSpec::ProcessCallback) [generator](ProcessingContext& processingContext) mutable {
-          usleep(100000);
-          size_t length = generator() % 10000;
-          auto data = processingContext.outputs().make<char>(Output{ "ITS", "RAWDATA", 0, Lifetime::Timeframe },
-                                                             length);
-          for (auto&& item : data) {
-            item = static_cast<char>(generator());
-          }
-        };
-      }
+  return (AlgorithmSpec::ProcessCallback)[generator](ProcessingContext & processingContext) mutable
```

Comment:
Here clang-format actually destroys the formatting instead of improving it. Can we keep it as it was or with // clang-format off ?